### DISCUSSION TITLE
[REQUEST] add donation with crypto

### DISCUSSION THREAD

**Discussion by damartripamungkas** [yao-pkg/pkg#22] [General]
add donation for this repo via crypto

**Comment by robertsLando** [comment]
ETH or BTC?

**Reply by damartripamungkas** [reply]
BSC

**Reply by robertsLando** [reply]
BNB: 0x5B580188CAF60cb2bD8336BADFf2A5f258221c2B

ETH: 0x5B580188CAF60cb2bD8336BADFf2A5f258221c2B

BTC: bc1q4d4gdgnqzaa03duv39lc69uqeq9cq2dp6t5s7h
